_config.yml

```diff
@@ -12,46 +12,61 @@ description: >
 
 # Build settings
 
+# If you clone the minima repo and build locally, use this setting.
 theme: minima
 
+# As of November 2023, GitHub Pages still uses minima 2.5.1 (https://pages.github.com/versions/). 
+# If you want to use the latest minima version on GitHub Pages, use the following setting and 
+# add a line comment on "theme: minima" above.
+#remote_theme: jekyll/minima
+
 plugins:
   - jekyll-feed
   - jekyll-seo-tag
 
 # Theme-specific settings
 
+# If you want to link only specific pages in your header, use this and add the path to the pages
+# in order as they should show up.
+header_pages:
+  - about.md
+
+# Set to `true` to show excerpts on the homepage.
+#show_excerpts: false
+
+# Minima-specific settings, which are only available from Minima 3.0 onward.
 minima:
+  # Minima skin selection.
+  # Available skins are:
+  # classic	           Default, light color scheme.
+  # dark	             Dark variant of the classic skin.
+  # auto	             Adaptive skin based on the default classic and dark skins.
+  # solarized-light	   Light variant of solarized color scheme.
+  # solarized-dark	   Dark variant of solarized color scheme.
+  # solarized	         Adaptive skin for solarized color scheme skins.
+  skin: classic
+  
   # Minima date format.
   # Refer to https://shopify.github.io/liquid/filters/date/ if you want to customize this.
-  #
-  # date_format: "%b %-d, %Y"
+  date_format: "%b %-d, %Y"
```

Comment:
I didn't see this change described in your PR description. Was it intentional to uncomment this? If not, would you mind recommenting it? Thanks!

Reply:
Yes, I forgot about that, sorry, it was late. The idea was, that the config file should reflect the sample site (https://jekyll.github.io/minima/). I just tested it, and it seems that the default date value is "%b %d, %Y" anyway (I couldn't find out what the minus before the day means, it seems to do nothing). I would therefor suggest, to comment the line again, but switch it to a format which is not the default (we could mention the default in the comment above).